heron/schedulers/src/java/org/apache/heron/scheduler/kubernetes/KubernetesContext.java

```diff
@@ -242,44 +253,51 @@ public static boolean getPodTemplateDisabled(Config config) {
     return getConfigItemsByPrefix(config, key);
   }
 
-  public static boolean getPersistentVolumeClaimDisabled(Config config) {
-    final String disabled = config.getStringValue(KUBERNETES_PERSISTENT_VOLUME_CLAIMS_CLI_DISABLED);
+  public static boolean getVolumesFromCLIDisabled(Config config) {
+    final String disabled = config.getStringValue(KUBERNETES_VOLUME_FROM_CLI_DISABLED);
     return "true".equalsIgnoreCase(disabled);
   }
 
   /**
    * Collects parameters form the <code>CLI</code> and generates a mapping between <code>Volumes</code>
    * and their configuration <code>key-value</code> pairs.
    * @param config Contains the configuration options collected from the <code>CLI</code>.
-   * @param isExecutor Flag used to collect CLI commands for the <code>executor</code> and <code>manager</code>.
+   * @param prefix Configuration key to lookup for options.
+   * @param isExecutor Flag used to switch CLI commands for the <code>Executor</code> and <code>Manager</code>.
    * @return A mapping between <code>Volumes</code> and their configuration <code>key-value</code> pairs.
    * Will return an empty list if there are no Volume Claim Templates to be generated.
    */
-  public static Map<String, Map<KubernetesConstants.VolumeClaimTemplateConfigKeys, String>>
-      getVolumeClaimTemplates(Config config, boolean isExecutor) {
+  @VisibleForTesting
+  protected static Map<String, Map<KubernetesConstants.VolumeConfigKeys, String>>
```

Comment:
Would it be worthwhile to mark this parameters passed in the function as `final`?

Reply:
Changes affected.